R/run.R

```diff
@@ -319,3 +318,30 @@ view_run_details <- function(run) {
                 caption = paste(unlist(details$warnings), collapse = "\r\n"),
                 options = list(dom = "t", scrollY = TRUE))
 }
+
+#' Show run details in viewer pane or browser
+#' @param run Run object
+#' @export
+view_run_details <- function(run) {
+  
+  # Run widget unless in Notebook VM
+  if (!grepl("rstudio-server", Sys.getenv("RS_RPOSTBACK_PATH")) &&
+      rstudioapi::isAvailable()) {
+    library(here)
+    path <- here("R", "app", "app.R")
+    
+    run_url <- run$get_portal_url()
+    assign("run_url", run_url, envir=globalenv())
+    
+    rstudioapi::jobRunScript(path, importEnv = TRUE)
```

Comment:
What if user is not on RStudio?

Reply:
I'll add more robust logic for that and job-tracking.